src/sentry/static/sentry/app/components/task/processListItem.jsx

```diff
@@ -65,7 +72,15 @@ ProcessListItem.propTypes = {
   processDefinitionKey: PropTypes.string.isRequired,
   processDefinitionName: PropTypes.string,
   count: PropTypes.number.isRequired,
-  tasks: PropTypes.arrayOf(PropTypes.shape({})),
+  tasks: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      count: PropTypes.number.isRequired,
+      taskDefinitionKey: PropTypes.string.isRequired,
+      flexWidth: PropTypes.number,
```

Comment:
I personally prefer having `view` related parameters either outside of the business model or within a specifically named variable (e.g. `viewState`). Just for readability.

Reply:
Fixed!